src/patternfly/assets/images/img_avatar-dark.svg

```diff
@@ -1,16 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 24.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 36 36" style="enable-background:new 0 0 36 36;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#212427;}
-	.st1{fill-rule:evenodd;clip-rule:evenodd;fill:#6A6E73;}
-	.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#4F5255;}
-</style>
-<rect class="st0" width="36" height="36"/>
-<path class="st1" d="M30.5,36c-0.4-3.9-1.3-9-2.9-11c-1.1-1.4-2.3-2.2-3.5-2.6s-1.8-0.6-6.3-0.6s-6.1,0.7-6.1,0.7
-	c-1.2,0.4-2.4,1.2-3.4,2.6C6.7,27,5.8,32.2,5.4,36H30.5z"/>
-<path class="st1" d="M17.7,20.1c-3.5,0-6.4-2.9-6.4-6.4s2.9-6.4,6.4-6.4s6.4,2.9,6.4,6.4S21.3,20.1,17.7,20.1z"/>
-<path class="st2" d="M13.3,36l0-6.7c-2,0.4-2.9,1.4-3.1,3.5L10.1,36H13.3z"/>
-<path class="st2" d="M22.7,36l0-6.7c2,0.4,2.9,1.4,3.1,3.5l0.1,3.2H22.7z"/>
-</svg>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width=".5in" height=".5in" version="1.1" viewBox="0 0 36 36">
```

Comment:
These have a width and height on the svg but the old ones did not - I'm not sure if that's good or bad or matters?

Reply:
It's possibly affecting https://patternfly-pr-6769.surge.sh/components/menus/menu-toggle#with-avatar-and-text

Reply:
Nice catch, I'd remove that. Pretty sure everywhere we use an avatar, we constrain the size if that's needed.